.github/scripts/template-zip-autogen.sh

```diff
@@ -33,6 +40,24 @@ for LANGUAGE in ${LANGUAGE_LIST[@]}; do
             exit -1
         fi
 
+        # Copy bot code to msgext-bot, except readme and images
+        rsync -avz --recursive --exclude "*.md" --exclude "*images/*" ./templates/bot/ ./templates/bot-msgext/
+        
+        # Generate code from Mustache templates
+        if [ ${SCOPE} == "bot" ]; then           
```

Comment:
NIT: put it outside of this loop, create a new loop for it.

```
BOT_FEATURES=(bot me)
for lan ...
  for feat ...
     
```

Reply:
ok

Reply:
I think put in the loop is ok, and can avoid extra for loop definition.